Feat: Improved Proxy URL support in Konnect (#6833)
Kong/docs.konghq.com: app/konnect/updates.md

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## February 2024

**Improved Proxy URL experience**

> : Customers are now able to configure and view their Proxy URLs directly from {{site.konnect_short_name}}, and use it them to connect to data planes directly.

Reviewer: ```suggestion
: You can now configure and view your Proxy URLs directly from {{site.konnect_short_name}}, and use them to connect to data planes directly.
```
